README.md

```diff
@@ -1,6 +1,6 @@
 # Functionless `λ<`
 
-[![npm version](https://badge.fury.io/js/functionless.svg)](https://badge.fury.io/js/functionless)
+[![npm version](https://badge.fury.io/js/functionless.svg)](https://badge.fury.io/js/functionless) [![Slack](https://img.shields.io/badge/slack-community-brightgreen?logo=slack)](https://functionless.slack.com/signup#/domain-signup)
```

Comment:
Does it render a badge?

Reply:
![image](https://user-images.githubusercontent.com/289213/173172231-52070099-0b2e-4724-9a08-1023d217cf87.png)